SPECS/mariner-rpm-macros/gen-ld-script.sh

```diff
@@ -30,7 +30,7 @@ case ${#NUM_DOT_SEPARATORS} in
     ;;
 esac
 
-MODULE_INFO_DIR="/usr/src/mariner/BUILD/"
```

Comment:
To be safe and consistent, let's also update this hard-coded path in the specs. I was able to find it used here:
- `ceph.spec`,
- `golang-1.17.spec`,
- `grub2.spec`, and
- `fish.spec` (that's in `SPECS-EXTENDED`).

Reply:
Good idea, I'll update these as well